Changes.md

```diff
@@ -0,0 +1,17 @@
+# Changes
+
+## August 25 2016
+
+Refactored the settings to start with `SNOOP_`.
+Settings that have been modified:
+
+- `MAX_TIKA_FILE_SIZE` --> `SNOOP_TIKA_MAX_FILE_SIZE`
+- `MSGCONVERT_SCRIPT` --> `SNOOP_MSGCONVERT_SCRIPT`
+- `SEVENZIP_BINARY` --> ... (same as above)
```

Comment:
Or simplify. Only list the old settings and the reader can figure out the new ones.